src/components/needLeadViewPage/needsViewTable.tsx

```diff
@@ -164,8 +169,8 @@ function NeedsViewTable() {
     <tr key={need.id}>
       <td className={styles.td}>{need.urgencyLevel}</td>
       <td className={styles.td}>{need.itemName}</td>
-      <td className={styles.td}>{need.createdAt}</td>
-      <td className={styles.td}>{need.createdAt}</td>
+      <td className={styles.td}>{need.authorId}</td>
+      <td className={styles.td}>{need.authorId}</td>
```

Comment:
why are you putting it twice ?

Reply:
This is an intermediary pull request. In the future I will be using the authorId to get the data for these td's (the author's name, the organization's name)